ui/webui/src/components/storage/InstallationDestination.jsx

```diff
@@ -147,69 +149,116 @@ const LocalStandardDisks = ({ idPrefix, onAddErrorNotification }) => {
         return <EmptyStatePanel loading />;
     }
 
+    const localDisksInfo = (
+        <Popover
+          bodyContent={
+              <div>{_(
+                  "Locally available storage devices (SATA, NVMe SSD, " +
+                  "SCSI hard drives, external disks, etc.)"
+              )}
+              </div>
```

Comment:
You don't need the `<div>`, instead just `bodyContent={_("text")}` is enough.

Reply:
Fixed.